cdc/owner/schema_test.go

```diff
@@ -72,6 +72,23 @@ func TestAllPhysicalTables(t *testing.T) {
 	require.Equal(t, schema.AllPhysicalTables(), expectedTableIDs)
 }
 
+func TestAllTableNames(t *testing.T) {
+	helper := entry.NewSchemaTestHelper(t)
+	defer helper.Close()
+	ver, err := helper.Storage().CurrentVersion(oracle.GlobalTxnScope)
+	require.Nil(t, err)
+	schema, err := newSchemaWrap4Owner(helper.Storage(), ver.Ver, config.GetDefaultReplicaConfig())
+	require.Nil(t, err)
+	require.Len(t, schema.AllTableNames(), 0)
+	// add normal table
+	job := helper.DDL2Job("create table test.t1(id int primary key)")
+	require.Nil(t, schema.HandleDDL(job))
+	require.Equal(t, schema.AllTableNames(), []model.TableName{{Schema: "test", Table: "t1"}})
```

Comment:
Better to change the variable sequence, since
```go
func Equal(t TestingT, expected interface{}, actual interface{}, msgAndArgs ...interface{})
```

Reply:
Oops! Thanks!